requirements.txt

```diff
@@ -1,23 +1,160 @@
 #
-# This file is autogenerated by pip-compile
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
>updated docker to python 3.11 (however, the libraries are generated by pip-compile available for python 3.9)

I'm sure the requirements*.txt files generated by python 3.9 and 3.11 are almost identical, but it would be great to know for certain. In a subsequent PR, could you try regenerating requirements*.txt with python 3.11?

Reply:
I could, but that would require putting Python 3.11 on the dev machine to use pip-compile tools, and it has been proven to be not a trivial task for me.